### ISSUE TITLE
Error in names(x) <- value :    'names' attribute [3] must be the same length as the vector [1]

### ISSUE THREAD

**Issue by meganrussell24** [sqjin/CellChat#437]
I followed the standard pipeline. This worked for one set of my data. I used the exact same code for my other datast. After the commands CM_EC_WT_cellchat <- identifyOverExpressedGenes(CM_EC_WT_cellchat)
CM_EC_WT_cellchat <- identifyOverExpressedInteractions(CM_EC_WT_cellchat) , I try to run df.net_all_CM_EC_WT <- subsetCommunication(CM_EC_WT_cellchat). I get the following error: Error in names(x) <- value : 
  'names' attribute [3] must be the same length as the vector [1]. I have gone through trying to figure out the error and read online, but I cant figure it out.

**Comment by sqjin** [comment]
@meganrussell24 Clearly, you have not run the key part of cellchat. You should at least run `computeCommunProb()`

**Comment by meganrussell24** [comment]
Yes, when I run computeCommunProb() I get the error: error in apply(prob, c(1, 2), by, group, sum) : dim(X) must have a positive length

**Comment by sqjin** [comment]
@meganrussell24 This is not expected. You should check your input data and the cellchat@data.signaling
